clients/frontend-authorization/src/FrontendAuthorizationClient.ts

```diff
@@ -19,7 +19,7 @@ export interface FrontendAuthorizationClient extends AuthorizationClient {
   signOut(requestContext?: ClientRequestContext): Promise<void>;
```

Comment:
Can we remove this entire interface now? Where is it still being used?

Reply:
It's being used in `iModelBank`/`iModelHub` Client functionality (mostly for testing it seems) but also in `ui-framework/oidc` (which I assume should be removed?)

If we're consuming the `AuthorizationClient` interface directly everywhere now, I only see `FrontendAuthorizationClient` as a way to organize classes under a shared interface. If `BrowserAuthorizationClient` remains the only auth client that implements it after the authorization client interface reorganization, however, `FrontendAuthorizationClient` probably doesn't need to exist for that class alone.

Reply:
Yea, the `ui-framework/oidc` usages are removed on master and just need to be merged. The iModelHub/iModelBank implementation is a bit of a strange one so I'm hoping that they can respond to the comment I left about the basic auth support.

I'd agree, get rid of `FrontendAuthorizationClient` and only keep `BrowserAuthorizationClient`.

Reply:
@calebmshafer, Don't we need an interface that allows you to signIn() and signOut() that can be used from something like SignIn.tsx (wherever that resides) that works for both the desktop and browser cases?

Reply:
@calebmshafer I don't know the answer to that, I have not worked in any of that part of the code. I did pull this branch and ui-test-app will not even startup properly, I can't even open a local file.